tests/test_pseudocone.py

```diff
@@ -317,7 +317,7 @@ def test_list_test_data_users_between_dates_date_format_error(mock_db_data, pseu
     request = pseudocone_pb2.ListTestDataUsersBetweenDatesRequest(limit=1,
                                                                   users=users,
                                                                   start_interaction_time="2018-01-01T00:00:26.318497Z",
-                                                                  end_interaction_time="201805-01T00:00:26.318497Z")
+                                                                  end_interaction_time="2018-60-01T00:00:26.318497Z")
```

Comment:
Would be nice to have a test that shows that both with and without the timezone info works.

Reply:
since all the tests used timezone info, I added an extra test that uses timestamps without timezone.